File: Python/백준/Bronze/1225. 이상한 곱셈/이상한 곱셈.py
aa=input().split(); a=int(aa[0]); b=int(aa[1]); sum1=0; sum=0
for i in range(len(aa[0])):
    sum1+=int(aa[0][i])
for i in range(len(aa[1])):
    sum+=sum1*int(aa[1][i])
print(sum)

File: C#/백준/Bronze/1225. 이상한 곱셈/이상한 곱셈.cs
using System;

namespace q{
    class w{
        static void Main(){
            string[] aa = Console.ReadLine().Split();
            int sum1=0; long sum=0;
            for (int i =0; i<aa[0].Length; i++){
                sum1+=(int)aa[0][i]-'0';
            }
            for(int i=0; i<aa[1].Length; i++){
                sum+=sum1*((int)aa[1][i]-'0');
            }
            Console.WriteLine(sum);
        }
    }
}